heat/core/tests/test_dndarray.py

```diff
@@ -402,6 +402,23 @@ def test_redistribute(self):
             with self.assertRaises(ValueError):
                 st.redistribute_(target_map=torch.zeros((2, 4)))
 
+    def test_reshape(self):
```

Comment:
these tests can be deleted. you can do the inline call in the manipulations tests. no need to replicate the tests

Reply:
Good to know. Thank you.